docroot/themes/humsci/humsci_basic/src/scss/partials/tools/_mixins.text.scss

```diff
@@ -200,13 +200,13 @@
 @mixin hb-body--small {
   font-weight: hb-theme-font-weight(regular);
   font-size: hb-calculate-rems(16px);
-  line-height: 122%;
+  line-height: var(--normal-line-height);
 }
 
 @mixin hb-body--medium {
   font-weight: hb-theme-font-weight(regular);
   font-size: hb-calculate-rems(18px);
-  line-height: 127%;
+  line-height: var(--normal-line-height);
 }
 
 @mixin hb-description {
```

Comment:
Let's refactor `hb-description`:
- Remove includes, add styles directly.
- Only alter the `font-size` in the media query.
- Review if the overrides for traditional are still necessary.